TLM/TLM/UI/Helpers/ExtUITabStrip.cs

```diff
@@ -120,7 +120,7 @@ public sealed class ExtUITabstrip : UITabstrip {
             UITabContainer tabContainer = optionsContainer.AddUIComponent<UITabContainer>();
             tabContainer.relativePosition = new Vector3(0, TAB_STRIP_HEIGHT);
             tabContainer.width = (orgOptsContainerWidth + paddingRight) - V_SCROLLBAR_WIDTH;
-            tabContainer.height = optionsContainer.height - (TAB_STRIP_HEIGHT - /*padding-top*/10 - /*margin-top*/ 10);
+            tabContainer.height = optionsContainer.height - (TAB_STRIP_HEIGHT + /*padding-top*/10 + /*margin-top*/ 10);
```

Comment:
More magic constants. Padding can be taken from UIComponent (UIPanel etc) fields called autolayoutPadding or something like that

Reply:
I will check that